Sources/StreamChatUI/CommonViews/AvatarView/CurrentChatUserAvatarView.swift

```diff
@@ -29,7 +29,6 @@ open class _CurrentChatUserAvatarView<ExtraData: ExtraDataTypes>: _Control, Them
     
     /// The view that shows the current user's avatar.
     open private(set) lazy var avatarView: ChatAvatarView = components
-        .currentUser
```

Comment:
This updates the `CurrentChatUserAvatarView` to use the common `components.avatarView`. If we want the avatar in `CurrentChatUserAvatarView` be customizable separately I'll create a new dedicated item